identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -335,3 +335,7 @@ QmFja3VwIENvZGUgRW5hYmxlZA__=Backup Code Enabled
 RW5hYmxlZCBBdXRoZW50aWNhdG9ycw__=Enabled Authenticators
 QmFja3VwIENvZGVz=Backup Codes
 RmFpbGVkIEJhY2t1cCBDb2RlIEF0dGVtcHRz=Failed Backup Code Attempts
+invitation.accepted=Invitation Accepted
+invitation.accepted.message=You now have access to your invited organization's applications. Reach out to your invited organization's administrator for information on available applications
```

Comment:
Do we need fulstop for the messages. If it is required, please add.

Reply:
No it was added in the JSP itself.
```
<p class="portal-tagline-description">
                            <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "invitation.accepted.message")%>.
```